client/app/scripts/components/app.js

```diff
@@ -46,6 +46,9 @@ class App extends React.Component {
       getTopologies(this.props.activeTopologyOptions, this.props.dispatch);
     }
     getApiDetails(this.props.dispatch);
+    if (this.props.contrastMode) {
+      this.props.toggleContrastMode(true);
+    }
```

Comment:
Does it not work for you as intented if you comment out these lines? For me it seems to work just fine still and I thought that was because it always gets triggered through the `route` action in `app-actions`.